store/aciinfo.go

```diff
@@ -100,6 +105,29 @@ func GetACIInfosWithName(tx *sql.Tx, name string) ([]*ACIInfo, bool, error) {
 	return aciinfos, found, err
 }
 
+// getAciInfosWithDockerID returns all the ACIInfos with a given docker ID.
+// found will be false if no aciinfo exists.
+func getACIInfosWithDockerID(tx *sql.Tx, dockerid string) ([]*ACIInfo, bool, error) {
+	var aciinfos []*ACIInfo
+	found := false
+	rows, err := tx.Query("SELECT * from aciinfo WHERE dockerid == $1", dockerid)
+	if err != nil {
+		return nil, false, err
+	}
+	for rows.Next() {
+		found = true
+		aciinfo := &ACIInfo{}
+		if err := aciinfoRowScan(rows, aciinfo); err != nil {
+			return nil, false, err
+		}
+		aciinfos = append(aciinfos, aciinfo)
+	}
+	if err := rows.Err(); err != nil {
+		return nil, false, err
+	}
+	return aciinfos, found, err
```

Comment:
`err` in this place could only hold `nil`, so it's IMO more readable to explicitly return `nil` instead of `err`.